gazebo_plugins/test/test_worlds/gazebo_ros_depth_camera.world

```diff
@@ -104,15 +104,23 @@
                             <height>480</height>
                             <format>R8G8B8</format>
                         </image>
-                        <clip near="0.01" far="100.0" />
-                        <save enabled="false" path="/tmp" />
-                        <depth_camera output="points" />
+                        <clip>
+                          <near>0.1</near>
+                          <far>100</far>
+                        </clip>
+                        <save enabled='0'>
+                          <path>__default__</path>
```

Comment:
Did you mean to change the path? Since the default is disabled, I think the whole `save` block could be removed.